README.md

```diff
@@ -5,6 +5,26 @@ Collection of performance tests designed to be run using [Gatling](https://gatli
 Currently the tests will ramp up from a low number of new users per second to a higher number of new users per second over a minute,
 then sustain that higher number of new users per second for a certain number of minutes.
 
+## Running the tests autonomously
+`./gradlew clean build` will create a 'fat jar' with all the dependencies.
```

Comment:
I always run `run_performance_tests_against_local.sh` when running the perfomance tests myself, is it not also worth mentioning this here too?